### ISSUE TITLE
Use Nexus where possible in jenkins-slave-base image build

### ISSUE THREAD

**Issue by michaelsauter** [opendevstack/ods-core#386]
For some (all?) dependencies we might be able to use Nexus. This will require configuring Nexus during the build process.

Originated in #378.

**Comment by clemensutschig** [comment]
yes yes yes ... we have all the data /and credentials to make this fly ...

**Comment by michaelsauter** [comment]
@clemensutschig Sorry to bother again, but ... I agree we have all the config/credentials available to make this fly. However, apart from the sonar-scanner-cli, we download everything from other sources than Maven. So configuring Nexus for this one dependency does not bring that much value in my eyes. Therefore I would take this off from 3.0 as it has low priority compared to things we have identified collectively that should be worked on for 3.0 and that already have found an owner. OK for you?
